setup.py

```diff
@@ -212,22 +213,46 @@ def get_llvm_package_info():
         elif arch == 'x64':
             vglibc = tuple(map(int, platform.libc_ver()[1].split('.')))
             vglibc = vglibc[0] * 100 + vglibc[1]
-            if vglibc > 228:
-                # Ubuntu 24 LTS (v2.39)
-                # Ubuntu 22 LTS (v2.35)
-                # Ubuntu 20 LTS (v2.31)
-                system_suffix = "ubuntu-x64"
-            elif vglibc > 217:
-                # Manylinux_2.28 (v2.28)
-                # AlmaLinux 8 (v2.28)
-                system_suffix = "almalinux-x64"
-            else:
-                # Manylinux_2014 (v2.17)
-                # CentOS 7 (v2.17)
-                system_suffix = "centos-x64"
-        else:
+
+            # NOTE: Update using `cmake/llvm-vglibc/update.sh` when new precompiled
+            # llvm builds are available
+            vglibc_dir = os.path.join(get_base_dir(), "cmake", "llvm-vglibc")
+
+            # Dynamically discover available system suffixes and
+            # read glibc version requirements by reading the folder
+            # Files should be named as <system-suffix>.txt (e.g., ubuntu-x64.txt)
+            system_configs = []
+            if os.path.exists(vglibc_dir):
+                for filename in sorted(os.listdir(vglibc_dir)):
```

Comment:
You can use a single loop to open and read the files, right? See current code seems longer than I expected.

Reply:
Now it's in a single loop